Tests/test_KEGG.py

```diff
@@ -58,6 +58,11 @@ def test_sample(self):
                           ('BRENDA, the Enzyme Database', ['1.1.1.1']),
                           ('CAS', ['9031-72-5'])])
         self.assertEqual(records[-1].entry, "2.7.2.1")
+        self.assertEqual(records[-1].__str__().replace(" ", "").split("\n")[:10],
```

Comment:
Rather than calling the magic method explicitly,  I would just use ``str(records[-1])`` here.